mailer/index.js

```diff
@@ -53,19 +36,37 @@ const EMAIL_CONTENT = {
     contentTop: `Congratulations on signing your offer. Details of your next steps are included in your offer letter and we are here to guide your process.`,
     contentBottom: `Meanwhile, click below to read our article on <a href="https://ballers.ng">5 Things Every Intending Homeowner Should Know</a>. We look forward to an enjoyable process.`,
   },
-  WELCOME_MESSAGE: {
-    subject: 'Welcome to BALLERS',
-    contentTop: `Congratulations you're officially a BALLer.`,
-  },
   RAISE_CONCERN: {
     subject: 'Concern Raised',
     contentTop: `A concern has been raised on your offer to [User name]. The question states [question].`,
     contentBottom: `Check your dashboard to resolve this concern.`,
   },
+  REFERRAL_INVITE: {
+    subject: 'BALLERS Invite',
+    contentTop: '[Referrer Name] just invited you to Become A LandLord on ballers.ng',
+    contentBottom: `Use their code [Referral code], or click the link below`,
```

Comment:
There are no provisions to use the referral code on the website, so lets use

```suggestion
    contentBottom: `Click on the link below to accept their invitation`,
```